capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -0,0 +1,21 @@
+document.body.prepend(dynamicButton);
+
+function createClassroom() {
+    var auth2 = gapi.auth2.getAuthInstance();
+    var name = "";
+    var email = "";
+    if (auth2.isSignedIn.get()) {
+        var name = auth2.currentUser.get().getBasicProfile().getName();
```

Comment:
Do we need a name variable for classrooms?

Reply:
This is to create the dummy user for now. The user dummy get its name here.